src/main/steps/tasklistresponse/proceedings/ProceedingDetailsMapper.test.ts

```diff
@@ -457,8 +457,8 @@ describe('ProceedingDetailsMapper', () => {
   });
 
   test('Should return value for providing a date for getDisplayDate', () => {
-    const retunrn_date = { day: undefined, month: undefined, year: 'June' };
-    expect(getDisplayDate(new Date())).toEqual(retunrn_date);
+    const retunrn_date = { day: '20', month: '03', year: '2023' };
```

Comment:
Remove this as it must gave been fixed already in 874PR

Reply:
Yes Vivek , this  is removed.